src/kernel_warnings.cpp

```diff
@@ -68,6 +68,7 @@ static std::vector<std::string> HipKernelWarnings()
         "-Wno-double-promotion",
         "-Wno-exit-time-destructors",
         "-Wno-extra-semi",
+        "-Wno-extra-semi-stmt",
```

Comment:
[Q] This looks like a workaround of some other problem (albeit non-critical). Can we resolve it right here?

Reply:
I'll need to regain access to a gfx94x machine before I can reassess.

Reply:
@cderb [Recommendation] Fix kernel build warnings right here and revert this change.

----
As usual, "recommendations" are up to the author discretion. The author can actually resolve or just mark as resolved -- no explanation needed.